docs/news/6192.fix

```diff
@@ -0,0 +1 @@
+Add fix for exception that could occur when using includes and top level members
```

Comment:
```suggestion
One less exception when using includes and top level members
```

Otherwise this will be rendered in the release notes as
"Fix: Add fix for exception that could occur when using includes and top level members"